models/public/anti-spoof-mn3/model.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+description: >-
+  "anti-spoofing-mn3" is an anti-spoofing binary classificator based on the MobileNetv3
```

Comment:
```suggestion
  "anti-spoof-mn3" is an anti-spoofing binary classificator based on the MobileNetv3
```

Reply:
@vladimir-dudnik 
> is there are any demo app associated with this model?

I have a Python demo. But as we discussed earlier, It's better to merge this model to the interactive_face_detection_demo pipeline. So, I'm working on this. I will create PR soon when finished.